README.md

```diff
@@ -7,30 +7,45 @@
 What is webpki?
 ==================
 
-webpki is a library that validates Web PKI (TLS/SSL) certificates. webpki
-is designed to provide a **full** implementation of the client side of the
-**Web PKI** to a diverse range of applications and devices,
-including embedded (IoT) applications, mobile apps, desktop applications, and
-server infrastructure. webpki is intended to not only be the best
-implementation of the Web PKI, but to also *precisely define* what the Web PKI
-is.
+webpki is a library that validates Web PKI (TLS/SSL) certificates. It's
+used by [Rustls](https://github.com/rustls/rustls) to handle certificate related
```

Comment:
Would we need to hyphenate "certificate-related"?

Reply:
Done.